index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="flex-container header">
+        <div class="flex-item">
+            <a href="#">Portfolio</a>
+        </div>
+        <div class="flex-item">
+            <a href="#">About</a>
+        </div>
+        <div class="flex-item">
+            <a href="#">Contact</a>
+        </div>
+    </div>
+    <div class="flex-container main">
```

Comment:
[OPTIONAL] I suggest that the `div` tag be changed to a `main` tag, so as to follow semantic HTML which in turn help search engines. 😃